docs/en/conf.py

```diff
@@ -57,6 +57,8 @@
     'sphinx_copybutton',
 ]  # yapf: disable
 
+autodoc_mock_imports = ['tensorrt']
```

Comment:
Why do we add this `autodoc_mock_imports`?

Reply:
Sphinx reported that it could not find `tensorrt` when building documentation.

![img1](https://user-images.githubusercontent.com/58739961/167386846-5772edc2-97b0-4dad-adec-483330bc5c9b.png)